README.md

```diff
@@ -1,2 +1,67 @@
 # docker-compose-gradle-plugin
-Simplifies usage of docker-compose for integration testing in Gradle environment.
+[![Build Status](https://travis-ci.org/avast/docker-compose-gradle-plugin.svg?branch=master)](https://travis-ci.org/avast/docker-compose-gradle-plugin)
+
+Simplifies usage of [Docker Compose](https://www.docker.com/docker-compose) for local development and integration testing in [Gradle](https://gradle.org/) environment.
+
+`composeUp` task starts the application and wait till all exported TCP ports are open (so till the application is ready). Then it exposes assigned host and ports of particular containers in `servicesInfos` property. `composeDown` task stops the application and removes the containers. These tasks can be executed manually or before/after tests.
+
+# Usage
+The plugin must be applied on project that contains `docker-compose.yml` file. It supposses that [Docker Engine](https://www.docker.com/docker-engine) and [Docker Compose](https://www.docker.com/docker-compose) are installed. The plugin calls these tools simply using [Project.exec()](https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:exec(groovy.lang.Closure)) method.
+
+If you are using [Java Gradle plugin](https://docs.gradle.org/current/userguide/java_plugin.html) then you might hook Docker Compose to the standard `test` task.
+```gradle
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.avast.gradle:docker-compose-gradle-plugin:$versionHere"
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'docker-compose'
+
+dockerCompose.isRequiredBy(test)
+
+test.doFirst {
+    // read current host and assigned port of service named 'web' (declared in docker-compose.yml)
```

Comment:
is service something known in java/gradle world? maybe module? Or how is it called?

For me this example is hard to understand...